packages/formatter/src/core/Tokenizer.ts

```diff
@@ -64,7 +64,8 @@ export default class Tokenizer {
   }
 
   createLineCommentRegex(lineCommentTypes) {
-    return new RegExp(`^((?:${lineCommentTypes.map(c => escapeRegExp(c)).join('|')})[^>]*?(?:\r\n|\r|\n|$))`, 'u');
+    const unMatchJSONOpeartors = '((?<!#)>|(?:[^>]))'; // Don't match if lineComment is # and immediately followed with >
```

Comment:
Please correct spelling on this and next line: unMatchJSONOpeartors -> unMatchJSONOperators

Reply:
I have done this myself.